stablehlo/tests/stablehlo_legalize_to_vhlo.0_9_0.mlir

```diff
@@ -1,6 +1,7 @@
 // RUN: stablehlo-opt --mlir-print-op-generic %s.bc | FileCheck %s
 // RUN: stablehlo-translate --deserialize %s.bc | stablehlo-translate --serialize --target=0.9.0 | stablehlo-opt --mlir-print-op-generic | FileCheck %s
 // RUN: diff <(stablehlo-translate --deserialize %s.bc | stablehlo-opt) <(stablehlo-opt %s)
+// RUN: diff <(stablehlo-translate --serialize --target=minimum %s | stablehlo-translate --deserialize | stablehlo-opt) <(stablehlo-opt %s)
```

Comment:
Am I right in thinking that the purpose of this RUN directive is orthogonal to this specific PR?

Reply:
Test that "minimum" as a serialization target works? Seems related to this PR?

Reply:
As discussed just now, let's delete this RUN directive?